tests/functional/replication/pauseResumeState.js

```diff
@@ -25,12 +25,16 @@ const repConfig = config.extensions.replication;
 const sourceConfig = {
     auth: { type: 'skip' },
 };
-const destConfig = {
-    auth: { type: 'skip', vault: 'skip' },
-    bootstrapList: repConfig.destination.bootstrapList,
-};
 const mConfig = config.metrics;
 
+function getDestSiteConfig(site) {
+    return {
+        auth: { type: 'skip', vault: 'skip' },
+        replicationEndpoint: repConfig.destination.bootstrapList
+            .find(endpoint => endpoint.site === site),
```

Comment:
probably a simpler way to rework this: `firstSite` and `secondSite` are already build by indexing into the bootstrapList, not sure there is a point to add some lookup code (and possily bug) in the tests